app/client/components/LoadingContentSpinner.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { PropsWithChildren } from "react";
+
+interface LoadingContentSpinnerProps {
+  isLoading: boolean;
+}
+
+export const LoadingContentSpinner = ({
+  isLoading,
+  children,
+}: PropsWithChildren<LoadingContentSpinnerProps>) => {
+  if (isLoading) {
```

Comment:
You could simply use ternary operator: `return isLoading ? (<span ...>) : (<>{children}</>)`

Reply:
Done